plutus-conformance/README.md

```diff
@@ -17,16 +17,57 @@ The tests currently covers or will cover:
 
 There is also an executable `add-test-output` for easy addition of test cases. Run 
 
-`cabal run add-test-output -- -h` for the manual.
+`cabal run add-test-output -- -h` 
 
-<!-- ## Testing alternative implementation
+for the manual.
 
-We provide a function  -->
-<!-- 
-## Untyped Plutus Core Evaluation
+## The Plutus Conformance Test Suite Library
+
+The library provides functions that users can import and run conformance tests with their own implementation.
+
+## Untyped Plutus Core Program Evaluation
+
+The `uplc-eval-test` test suite ensures conformance of evaluation of untyped plutus core programs. The expected output may contain:
+
+1. "parse error"
+
+The input files are expected to have the concrete syntax of our parser. The expected output will show "parse error" when the parser fails to parse the input file.
+
+2. "evaluation error"
 
-The `uplc-eval-test` test suite ensures that the input untyped plutus core programs evaluate to the expected output. The expected output is obtained by evaluating the programs using the CEK machine.
+If the input file cannot be evaluated, the expected output will show "evaluation error".
```

Comment:
I'd say "if evaluation fails with an error" as "cannot be evaluated" is somewhat ambiguous.